### PR TITLE
test: fix config error in test cases

### PR DESCRIPTION
**Pull request by monkeyDluffy6017**
### Description

Some yaml config in test cases are incorrect because we don't handle the result of schema validatation.

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: t/tars/discovery/tars.t [hunk 1]
```diff
@@ -41,8 +41,8 @@ discovery:
       database: db_tars
       user: root
       password: tars2022
-    full_fetch_interval: 3
-    incremental_fetch_interval: 1
+    full_fetch_interval: 90
```
Comment (zhixiongdu027) [inline t/tars/discovery/tars.t:44]: Please see [link](https://github.com/apache/apisix/pull/6599#discussion_r827587517)
   >     "full_fetch_interval: 3"
   >     "incremental_fetch_interval: 1"
   I think we should keep it as it is

Comment (monkeyDluffy6017) [reply t/tars/discovery/tars.t:44]: The APISIX will fail before modify the schema, maybe we should figure out other way to reduce the interval.

Comment (monkeyDluffy6017) [reply t/tars/discovery/tars.t:44]:
```   
 -- fill the default value by the schema
    schema.validate(default_conf)
```
We didn't handle the result of schema validation before, that's why your solution work, we need to throw error now

Comment (zhixiongdu027) [reply t/tars/discovery/tars.t:44]: maybe we should find another way
@spacewander @monkeyDluffy6017

Comment (spacewander) [reply t/tars/discovery/tars.t:44]: > The APISIX will fail before modify the schema, maybe we should figure out another way to reduce the interval.

@monkeyDluffy6017 
Interesting. The modification is done in the init_by_lua phase. It is strange that the schema validation happens before it. Could you dig deeper into it?

Comment (monkeyDluffy6017) [reply t/tars/discovery/tars.t:44]: Sure, i will check this later

File: apisix/core/config_local.lua [hunk 2]
```diff
@@ -65,7 +65,10 @@ function _M.local_conf(force)
     end
 
     -- fill the default value by the schema
-    schema.validate(default_conf)
+    local ok, err = schema.validate(default_conf)
+    if not ok then
+        ngx.log(ngx.ERR, "======================== levy, err: ", err)
```
Comment (tokers) [inline apisix/core/config_local.lua:70]: levy?

Comment (monkeyDluffy6017) [reply apisix/core/config_local.lua:70]: It will be changed to error(),  I just want to find out how many test cases will failed and fix them.

Comment (monkeyDluffy6017) [reply apisix/core/config_local.lua:70]: I have changed this pr to draft.

File: t/APISIX.pm [hunk 3]
```diff
@@ -502,6 +503,8 @@ _EOC_
 
     require "resty.core"
 
+    $extra_init_by_lua_start
```
Comment (spacewander) [inline t/APISIX.pm:506]: Could we just move $extra_init_by_lua to this place?

Comment (spacewander) [reply t/APISIX.pm:506]: Ping @monkeyDluffy6017

Comment (monkeyDluffy6017) [reply t/APISIX.pm:510]: Done

File: t/tars/discovery/tars.t (hunk 1 above)
Comment (monkeyDluffy6017) [reply t/tars/discovery/tars.t]: The modification should be put before apisix = require("apisix")，the validation is called in that.

**Review by tzssangglass — APPROVED** [review]

**Comment by monkeyDluffy6017** [comment]
test

File: apisix/core/config_local.lua (hunk 2 above)
Comment (monkeyDluffy6017) [reply apisix/core/config_local.lua]: Done

File: t/APISIX.pm [hunk 4]
```diff
@@ -493,6 +493,11 @@ _EOC_
 
     $block->set_value("main_config", $main_config);
 
+    # The new directive is introduced here to modify the upper and
```
Comment (spacewander) [inline t/APISIX.pm:496]: We can modify the schema, not only the upper and lower limits

File: t/APISIX.pm [hunk 5]
```diff
@@ -493,6 +493,11 @@ _EOC_
...
+    # lower limits of the schema before apisix validate in require("apisix")
+    # Todo: merge extra_init_by_lua_start and extra_init_by_lua
```
[the remark this answers is not in the document]
Comment (monkeyDluffy6017) [reply t/APISIX.pm:498]: https://github.com/apache/apisix/discussions/7988

[the remark this answers is not in the document]
Comment (monkeyDluffy6017) [reply t/APISIX.pm:498]: https://github.com/apache/apisix/discussions/7988

File: t/APISIX.pm (hunk 4 above)
Comment (monkeyDluffy6017) [reply t/APISIX.pm]: Done

**Review by spacewander — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**